Strapi don't check if current directory is actually a strapi project

On requesting start-up of my strapi application, before anything, strapi don't check if the current directory is a strapi project.. 
I type this command in the wrong directory `NODE_ENV=production strapi start`.

A little enhancement may be ?

Here my logs :
```
ERROR (17578 on vps46677): Cannot find module '/opt/config/environments/development/server.json'
    Error: Cannot find module '/opt/config/environments/development/server.json'
        at Function.Module._resolveFilename (module.js:538:15)
        at Function.Module._load (module.js:468:25)
        at Module.require (module.js:587:17)
        at require (internal/module.js:11:18)
        at Command.module.exports (/usr/lib/node_modules/strapi/bin/strapi-start.js:45:20)
        at Command.listener (/usr/lib/node_modules/strapi/node_modules/commander/index.js:315:8)
        at emitTwo (events.js:126:13)
        at Command.emit (events.js:214:7)
        at Command.parseArgs (/usr/lib/node_modules/strapi/node_modules/commander/index.js:653:12)
        at Command.parse (/usr/lib/node_modules/strapi/node_modules/commander/index.js:475:21)
```

--- a/packages/strapi/bin/strapi-start.js
+++ b/packages/strapi/bin/strapi-start.js
@@ -27,6 +27,11 @@ const { cli, logger } = require('strapi-utils');
  */
 
 module.exports = function() {
+  // Check that we're in a valid Strapi project.
+  if (!cli.isStrapiApp()) {
+    return logger.error('This command can only be used inside a Strapi project.');
+  }
+  
   try {
     const strapi = function () {
       try {

